.travis.yml

```diff
@@ -1,38 +0,0 @@
-# Install latex, and the various packages used by the start_paper Makefile:
```

Comment:
It was only testing the Note about start_paper, not actually running
start_paper. It's a good idea to have start_paper under continuous
integration, but I think this can be issued for the future. (I'll try to
remember to keep a copy of the .travis.yml file.)